contents.json

```diff
@@ -5444,6 +5444,11 @@
       "category": "xml",
       "description": "XMLEncoder & XMLDecoder based on Codable protocols from the standard library",
       "homepage": "https://github.com/MaxDesiatov/XMLCoder"
+    }, {
+      "title": "Carbon",
+      "category": "utility",
```

Comment:
Hi @ra1028, let's change the category to `form`. It seems to be more appropriate than `utility`.

Reply:
Hi @vpeschenkov 
Thanks for pointed out.
But, the feature to create form UI is just part of example. If `utility` feels inappropriate, how about changing category to `uicollectionview` or `uitableview` ?

Reply:
@ra1028 We can add a project only to one category, but this framework works with both `UICollectionView` &`UITableView`. Futhermore, in general meaning this framework builds UI, that is pretty close to the meaning of `Form` than just working with `UICollectionView` &`UITableView`, because it allows to build UI by using/extending predefined classes.

Reply:
Ah, make sense.